php/src/ContainerDefinitionFetcher.php

```diff
@@ -40,12 +41,25 @@ public function GetContainerById(string $id): Container
         throw new \Exception("The provided id " . $id . " was not found in the container definition.");
     }
 
+    private function validateJson(array $data): void {
+        // Validate against json schema
+        $validator = new Validator;
+        $validator->validate($data, [file_get_contents(__DIR__ . '/../containers.json')]);
+        if (!$validator->isValid()) {
+            error_log("JSON does not validate. Violations:");
+            foreach ($validator->getErrors() as $error) {
+                error_log(printf("[%s] %s\n", $error['property'], $error['message']));
+            }
+        }
+    }
+
     /**
      * @return array
      */
     private function GetDefinition(bool $latest): array
     {
         $data = json_decode(file_get_contents(__DIR__ . '/../containers.json'), true);
+        $this->validateJson($data);
```

Comment:
possible that this needs to be validated as object and not as array, so would simplay need to get the file and not json_decode it...